upgradability/forward-proxy/prime_arithmetic_lib_v1/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "prime_arithmetic_lib_v1"
+version = "0.1.0"
+authors = ["Cardinal Cryptoghraphy"]
+edition = "2021"
+
+[dependencies]
+ink_primitives = { version = "3", default-features = false }
```

Comment:
hmm, maybe use the same version of ink libs in all the contracts, it's fine I guess to use 3 as major version